doc/design/libraries.md

```diff
@@ -17,7 +16,7 @@
 
 ## Conventions
 
-- Most libraries are internal libraries and have APIs which are completely unstable! There are few or no restrictions on backwards compatibility or rules about external dependencies. Exceptions are *libbitcoin_consensus* and *libbitcoin_kernel* which have external interfaces documented at [../shared-libraries.md](../shared-libraries.md).
+- Most libraries are internal libraries and have APIs which are completely unstable! There are few or no restrictions on backwards compatibility or rules about external dependencies. Exceptions are *libbitcoin_consensus* and *libbitcoin_kernel*, which, at some future point, will have documented external interfaces.
```

Comment:
In the future, `libbitcoin_kernel` will have "documented external interface", but not `libbitcoin_consensus`, right?